peoplehr/tests/test.bal

```diff
@@ -236,3 +236,121 @@ function testCheckDuplicateApplicant() returns error? {
     OperationStatus response = check baseClient->checkDuplicateApplicant(applicantInformation);
     log:printInfo(response.toString());
 }
+
+@test:Config {enable: false}
```

Comment:
Any reason for disabling all the tests?

Reply:
Yes, Our PeopleHR test account has been expired. Therefore, these were tested using a sandbox belongs to the DigiOps team. However, We have already requested to extend the account trail-period. Will hope to enable the account ASAP we get the account.